Thesis.tex

```diff
@@ -222,11 +225,85 @@ \subsection{Reasonable and non-discriminatory licensing}
+
+As a last argument, the author wants to mention copyright trolls like Prenda Law or Malibu Media, that are suspected (and in the case of Prenda Law even convicted) of seeding porn to bit torrent themselves to be able to exact settlement payments in such a way that even people not having downloaded that content would rather pay than having to face a costly lawsuit, alleging them to be porn users. While the author has selected the most extreme case as illustration, there are numerous other ways that copyright trolls exact payments in other areas of arts, very often without a real basis, because the defendants cannot hope to bring up the money to defend themselves against a corporation with deep pockets.
+
+\subsection{Addressing Against IP's points}
+
+\subsection{Adressing Pro IP's points}
+
+\section{Copyright in the context of the creation of software}
+\label{sec:CopySoft}
+
+\subsection{Pro IP}
+
+\subsection{Against IP}
+In \ref{ssec:SLic}, the fact that terms limiting the freedom of the user can be required to get a license for a product, with the example of Office 365 that requires a yearly payment. In fact, Office 365 ceases working after a certain period without subscription. Imagine a person having lost his job and not being able to pay that subscription, having his outdated CV only in electronic form, as a Word document. He would, in the time of his greatest need, be able to update his CV properly, and being not able to pay the internet provider, this person might not be able to download a free text processing software, even if he knew about one. 
+
+The author of this chapter is a firm believer in open source software. Innovation can no longer exist in a vacuum, because so many things already exist. And innovation is important for the advancement of human culture. The digital age would not have been realized the way it has been were it not for Google. Google on the other hand would not have succeeded were it for OSS. The costs of proprietary operating systems for server farms the size of Google's would have made the rise of the search engine impossible. Another reason that OSS is to be preferred to closed source software (called CSS from here on) as described in \ref{ssec:SLic} is the data that is created and managed by the software. If a software publisher goes out of business, the software will no longer be updated or maintained. Within two or three operating system lifecycles, it is no longer possible to run that software at all, thus making all the data created worthless, because it is no longer readable. While standards can weaken this effect, not every software adheres to a standard, so the binary file formats are useless. Imagine using a comfortable database application for managing your family photos, and you lose the photos, just because the software is no longer available to read the archive. Such a scenario is realistic, but must not be.
```

Comment:
Such a scenario is realistic, but _should_ not be.
At least that's what I think you meant =)